File: src/main/java/kr/ac/uos/smartcity/model/Notice.java
package kr.ac.uos.smartcity.model;

import lombok.AllArgsConstructor;
import lombok.Data;

@AllArgsConstructor
@Data
public class Notice {
    private String id;
    private String title;
    private String registered;
    private String readCount;
    private String content;
    private String attachmentUrl;
    private String attachmentName;
}

File: src/frontend/src/types/Notice.ts
export class Notice {
    constructor(
        public id: number = 0,
        public title: string = '',
        public registered: string = '',
        public readCount: number = 0,
        public content: string = '',
        public attachment?: File,
        public attachmentName?: string,
        public attachmentUrl?: string) {
    }
}